packages/skill-api/src/core/services/convertkit/convertkit-tag-purchase.ts

```diff
@@ -41,6 +41,9 @@ async function getSanityProduct(productId: string) {
 }
 
 export async function convertkitTagPurchase(email: string, purchase: Purchase) {
+  // skip CK tagging in development
+  if (process.env.NODE_ENV === 'development') return
```

Comment:
```suggestion
  if (process.env.SKIP_CK_TAGGING === 'true') return
```

I'd prefer if this was explicit because the broad strokes makes for weird debugging sometimes